apps/monarch/monarch-backend/src/dynamodb.ts

```diff
@@ -131,10 +139,19 @@ export async function updatePractitioner(req: Request): Promise<Practitioner> {
   // Prefix with ':' to use as expression attribute values
   const practitionerData = practitionerDataFromBody(req, practitionerSchema.omit({ uuid: true }), ':');
 
+  const newGeocode = await extractGeocode(practitionerData[':businessLocation']);
```

Comment:
Whenever a practitioner is updated, we should update its geocode as well to reflect a (potentially) new address